category_theory/limits/cones.lean

```diff
@@ -45,6 +45,28 @@ lemma cocones_obj (X : C) : F.cocones.obj X = (F ⟹ (const J).obj X) := rfl
 
 end functor
 
+section functoriality
+variables (J C)
+
+def cones : (J ⥤ C) ⥤ Cᵒᵖ ⥤ Type _ :=
+{ obj := functor.cones,
+  map := λ F G f, whisker_left _ $ whisker_left _ (yoneda.map f) }
+
+def cocones : (J ⥤ C)ᵒᵖ ⥤ C ⥤ Type _ :=
+{ obj := functor.cocones,
+  map := λ F G f, whisker_left _ (coyoneda.map f) }
```

Comment:
Fill in the underscores for `Type _`: sometimes it's helpful to be told what universes things end up in.

Possibly filling in the underscores for `whisker_left _` would be helpful, for the sake of making the construction clearer, but I don't really mind here.

Reply:
I added the explicit type universe. I think the first argument to `whisker_left` is not very helpful, so I left it out.